src/ui_components/search_input.js

```diff
@@ -35,6 +35,10 @@ export default class SearchInput {
     window.__searchInput.searchInputHandle.blur()
   }
 
+  static setInputValue(value) {
+    document.getElementById('search').value = value
```

Comment:
for maintainability, we could use the existing suggest `setValue` method
```suggestion
    window.__searchInput.suggest.setValue(value)
```